go.mod

```diff
@@ -8,6 +8,8 @@ require (
 	golang.org/x/term v0.23.0
 )
 
+// replace fortio.org/cli => ../cli
```

Comment:
Why this?

Reply:
I was testing https://github.com/fortio/cli/pull/38